doc/fundamentals/append_overwrite.rst

```diff
@@ -1,571 +1,364 @@
-will, as you’ve probably guessed, overwrite the file, rather than append
-it.
-
-::
-
-   $ pachctl put file voterData@master -f OHVoterData.csv --overwrite
-   $ pachctl list file voterData@master
-   COMMIT                           NAME             TYPE COMMITTED    SIZE
-   8560235e7d854eae80aa03a33f8927eb /OHVoterData.csv file 1 second ago 153.8KiB
-   $ pachctl put file voterData@master -f OHVoterData.csv --overwrite
-   $ pachctl list file voterData@master
-   COMMIT                           NAME             TYPE COMMITTED    SIZE
-   4876f99951cc4ea9929a6a213554ced8 /OHVoterData.csv file 1 second ago 153.8KiB
-
-Deduplication
-~~~~~~~~~~~~~
-
-Pachyderm will deduplicate data loaded into input repositories. If you
-were to load another file that hashed identically to “OHVoterData.csv”,
-there would be one copy of the data in Pachyderm with two files’
-metadata pointing to it. This works even when using the append behavior
-above. If you were to put a file named OHVoterData2004.csv that was
-identical to that first ``put file`` of OHVoterData.csv, and then update
-OHVoterData.csv as shown above, there would be two sets of bits in
-Pachyderm:
-
--  a set of bits that would be returned when asking for the old branch
-   of OHVoterData.csv & OHVoterData2004.csv and
--  a set of bits that would be appended to that first set to assemble
-   the new, master branch of OHVoterData.csv.
-
-This deduping happens with each file as it’s added to Pachyderm. We
-don’t do deduping within the file (“intrafile deduplication”) because
-this system is built to work with any file type.
-
-.. important::  Pachyderm is smart about keeping the minimum set of bits in the object store and
-                assembling the version of the file you (or your code!) have asked for.
-
-Use cases for large datasets in single files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Splitting large files in Pachyderm
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Unlike a system like Git,
-which expects almost all of your files to be text,
-Pachyderm does not do intra-file diffing
-because we work with any file type:
-
-* text
-* JSON
-* images
-* video
-* binary
-* and so on…
-
-Pachyderm diffs content at the per-file level.
-Therefore,
-if one bit in content of a file changes,
-Pachyderm sees that as a new file.
-Similarly,
-Pachyderm can only distribute computation at the level of a single file;
-if your data is only one large file,
-it can only be processed by a single worker.
-
-Because of these reasons, it’s pretty common to break up large files
-into smaller chunks. For simple data types, Pachyderm provides the
-``--split`` flag to ``put file`` to automatically do this for you. For
-more complex splitting patterns (e.g. ``avro`` or other binary formats),
-you’ll need to manually split your data either at ingest or with a
-Pachyderm pipeline.
-
-Split and target-file flags
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Adjusting Data Processing by Splitting the Data
```

Comment:
There are important changes in this file. I did not rename it and did not add to how-tos yet so that the diff is visible.